.github/workflows/daily.yml

```diff
@@ -32,12 +32,29 @@ jobs:
         # 1. RUSTSEC-2021-0020 - Not impacted, see #7723
         # 2. RUSTSEC-2020-0146 - Not impacted, see #7826
         run: |
-          cargo audit --ignore RUSTSEC-2021-0020 --ignore RUSTSEC-2020-0146 >> $MESSAGE_PAYLOAD_FILE
-      - uses: ./.github/actions/slack-file
+          cargo audit --color never --ignore RUSTSEC-2021-0020 --ignore RUSTSEC-2020-0146 > $AUDIT_SUMMARY_FILE
+      - uses: actions/github-script@v3
+        if: ${{ failure() }}
+        id: set-content
         with:
-          webhook: ${{ secrets.WEBHOOK_AUDIT }}
-          payload-file: ${{ env.MESSAGE_PAYLOAD_FILE }}
+          script: |
+            const fsp = require('fs').promises
+            const summary = await fsp.readFile('${{ env.AUDIT_SUMMARY_FILE }}', 'utf-8')
+            const job_url = `https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}`
+            return `Found RUSTSEC in dependencies in job ${job_url}
+            \`\`\`
+            ${summary}
+            \`\`\`
+            `
```

Comment:
torn on this, is it better than shell, or worse to use github-script action from future maintenance and reusability?

Reply:
This was to avoid escaping special chars in multiline env var using set-output. 

Just found out toolkit now supports env file,  https://github.com/actions/toolkit/blob/main/docs/commands.md#environment-files.  Going to stick with shell.